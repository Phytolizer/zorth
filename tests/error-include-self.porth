include "./error-include-self.porth"
