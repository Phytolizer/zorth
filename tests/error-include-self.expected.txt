:i returncode 1
:b stdin 0

:b stdout 0

:b stderr 79
./error-include-self.porth:1:1: ERROR: include exceeded expansion limit (1000)

